pkg/authprovider/authx/testData/example-auth.enc.yaml

```
@@ -0,0 +1,89 @@
+      token: ENC[AES256_GCM,data:X7qf4A==,iv:UzXvYgxOJYRk+SRVqExF5qRVTmMsv4k6IBnm+nBAcK0=,tag:fe3J5vH6AuTCZY+QGZ0ntA==,type:str]
+# dynamic secrets (powered by nuclei-templates)
+dynamic:
+    - template: /path/to/wordpress-login.yaml
+      variables:
+        - name: username
+          value: ENC[AES256_GCM,data:WFGVf0SB,iv:/Tt247XYHPwATL19euhGrXy46tUwtQgqOh18Ikc3CVc=,tag:cAVTwY0ztL6CgwRCWtTy5Q==,type:str]
+        - name: password
+          value: ENC[AES256_GCM,data:DUA8UHq+oOQUHaU3Ww==,iv:S8GO5uznYOsHfZJh5z0ZYQnau2/Fyb9bRGXyPEet4Pw=,tag:wc8KT2ZDuUkpdtoxCKPFSQ==,type:str]
+      type: Cookie
+      domains:
+        - localhost:8080
```

yamllint (1.35.1)

[warning] wrong indentation: expected 10 but found 8

(indentation)